### DISCUSSION TITLE
When is Thymeleaf 3.1.0 due?

### DISCUSSION THREAD

**Discussion by snicoll** [thymeleaf/thymeleaf#925] [General]
Hey,

Spring Boot is relying on RC2 and we're releasing the GA next week. It's a bit odd that our GA provides dependency management for a non-GA third-party dep. Can you share when Thymeleaf 3.1.0 GA is due? 

Thanks!

**Comment by danielfernandez** [comment]
Just after Spring Framework 6.0.0 is published tomorrow (according to [this milestone](https://github.com/spring-projects/spring-framework/milestone/305)).

Thymeleaf 3.1 has dependencies on Spring Framework, Spring Security and Spring Webflow, and I had basically the same non-GA issue :). The idea is that Thymeleaf 3.1.0 is released with a dependency on Spring Framework 6.0.0 GA due to it being the _main_ Spring dependency, although it will have to keep a dependency on a Spring Security RC and a Spring Webflow milestone.

**Comment by snicoll** [comment]
Awesome, thanks!
